app/assets/javascripts/govuk_publishing_components/components/reorderable-list.js

```diff
@@ -0,0 +1,108 @@
+//= require sortablejs/Sortable.js
```

Comment:
As this file comes from the node_modules directory we'll need to include that in packaging of the gem: https://github.com/alphagov/govuk_publishing_components/blob/fba876a49e6512e339b4c27719cd6f1b017db364/govuk_publishing_components.gemspec#L17

Reply:
done